### PR TITLE
Resolves the issues of crud & entity generation when the entity is namespaced

### PR DESCRIPTION
**Pull request by starx**

### CODE REVIEW

**Comment by fabpot** [comment]
Change of the execution flag on files should be reverted (0644 instead of 0755)

File: Resources/skeleton/crud/actions/edit.php.twig [hunk 1]
```diff
@@ -17,7 +17,7 @@
     {
 {% block method_body %}
         $deleteForm = $this->createDeleteForm(${{ entity_singularized }});
-        $editForm = $this->createForm('{{ namespace }}\Form\{{ entity_class }}Type', ${{ entity_singularized }});
+        $editForm = $this->createForm('{{ namespace }}\Form\{% if entity_namespace is not empty %}{{ entity_namespace }}\{% endif %}{{ entity_class }}Type', ${{ entity_singularized }});
```
Comment (fabpot) [inline Resources/skeleton/crud/actions/edit.php.twig]: Where does that `entity_namespace`variable come from?

Comment (starx) [inline Resources/skeleton/crud/actions/edit.php.twig]: @fabpot On `generateControllerClass()` of `DoctrineCrudGenerator`.

**Comment by starx** [comment]
@fabpot File mode have been updated.

**Comment by HeahDude** [comment]
Thank you @starx for working on this :) However I'm gonna close here in favor of #508 and #509 which are a bit different and the most important add some tests to prevent regressions.

Thank you for understanding.

**Pull request closed without merging**